modules/nw-externalip-about.adoc

```diff
@@ -0,0 +1,178 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/configuring-externalip.adoc
+
+[id="nw-externalip-about_{context}"]
+= About ExternalIP
+
+For non-cloud environments, {product-title} supports the assignment of external IP addresses to a Service `spec.externalIPs` field through the *ExternalIP* facility.
+This exposes an additional virtual IP address, assigned to the Service, that can be outside the service network defined for the cluster.
+A Service configured with an external IP functions similarly to a Service with `type=NodePort`, allowing you to direct traffic to a local node for load balancing.
+
+You must configure your networking infrastructure to ensure that the external IP address blocks that you define are routed to the cluster.
+
+{product-title} extends the ExternalIP functionality in Kubernetes by adding the following capabilities:
+
+- Restrictions on the use of external IP addresses through a configurable policy
+- Allocation of an external IP address automatically to a Service upon request
+
+By default, only a user with `cluster-admin` privileges can create a Service with `spec.externalIPs[]` set to IP addresses defined within an external IP address block.
+
+[CAUTION]
+====
+Disabled by default, use of ExternalIP functionality can be a security risk, because in-cluster traffic to an external IP address is directed to that Service.
+This could allow cluster users to intercept sensitive traffic destined for external resources.
+====
+
+[IMPORTANT]
+====
+This feature is supported only in non-cloud deployments.
+For cloud deployments, use the load balancer services for automatic deployment of a cloud load balancer to target the endpoints of a service.
+====
+
+You can assign an external IP address in the following ways:
+
+Automatic assignment of an external IP::
+{product-title} automatically assigns an IP address from the `autoAssignCIDRs` CIDR block to the `spec.externalIPs[]` array when you create a Service CR object.
```

Comment:
Three corrections
1 - add a reference below to how administrators can set `autoAssignCIDRs` - those aren't on the Service object
2 - Service isn't a custom resource (CR).
3 - This only happens when you create a service with `spec.type = LoadBalancer`. Otherwise, we won't automatically allocate an IP.

Reply:
@squeed,

1. We have a weird restriction where we can't link internally to other sections, so instead I added:
   > Automatic assignment is disabled by default and must be configured by a cluster administrator as described in the following section.

   Or did you mean something else?

1. Oops, thanks. That's a legacy from when I didn't know anything about k8s extension mechanisms.

1. Oops, for some reason I removed that language; I've added it back.